src/pages/login.vue

```diff
@@ -6,7 +6,7 @@
             </h1>
             <div class="oc-login-card-body">
                 <h3 class="oc-login-card-title">
-                    <span v-translate>Welcome to</span> {{ configuration.theme.general.name }}
+                  <span v-text="$_welcomeMessage" />
```

Comment:
Want to try `<translate :translate-params="{productName: $_productName}">Welcome to %{productName}</translate>` here and only define `$_productName` as computed property? :-)